__tests__/app.test.js

```diff
@@ -41,3 +41,44 @@ describe("/ap/categories", () => {
     });
   });
 });
+
+describe('/api/reviews/:review_id', () => {
+  describe('get request ', () => {
+    test('returns the review of the id given', () => {
+      return request(app)
+      .get('/api/reviews/1')
+      .expect(200)
+      .then(({body}) =>{
+        expect(body.review).toEqual({
```

Comment:
I would still use `.toMatchObject` here to allow for more properties in future - but the exact assertions of each property is spot on!